seminar_5/task_2.py

```diff
@@ -0,0 +1,12 @@
+"""Напишите однострочный генератор словаря, который принимает на вход три списка одинаковой длины:
+имена str, ставка int, премия str с указанием процентов вида «10.25%». В результате получаем словарь
+с именем в качестве ключа и суммой премии в качестве значения. Сумма рассчитывается как ставка умноженная
+на процент премии"""
+
+names = ["Алиса", "Василий", "Петруша"]
+rates = [22, 40, 80]
+bonuses = ["10.25%", "5.50%", "8.75%"]
+
+result_dict = {name: rate * float(bonus.strip('%')) / 100 for name, rate, bonus in zip(names, rates, bonuses)}
```

Comment:
Верно, но можно без метода `strip`
```suggestion
result_dict = {name: rate * float(bonus.[:-1]) / 100 for name, rate, bonus in zip(names, rates, bonuses)}
```